src/PostListingDisplay/components/InputField.jsx

```diff
@@ -27,9 +37,9 @@ function setDefault(type){
                 day = '0' + day;
             } 
             if(month < 10) {
-                month = '0' + month;
+                month = '0' + (month+1);
```

Comment:
This seems wrong - if month values are returned 0-11, you'll print month 9 as 010, and month 10 as 10?

Reply:
You're right. I put the "+1" in the wrong spot. Moved it to where month first gets initialized.